### DISCUSSION TITLE
display hierarchy upside down

### DISCUSSION THREAD

**Discussion by brno32** [airbnb/visx#1089] [Q&A]
I've got something based off of the example here https://airbnb.io/visx/dendrograms

Is there a way to flip it upside-down? e.g., show $ at the bottom and the children above it?

**Comment by williaster** [comment]
Hey @brno32  👋 , yes this should be straight forward! 

You can simply update the `<Cluster />` `size` prop from `[width, height]` to `[width, -height]`. Y-coordinates in svg start at 0 at the top of the chart and are positive going down, so this will result in node positions from `0` (top of the chart) to `-height` px _above_ the chart (thus getting clipped). 

<img src="https://user-images.githubusercontent.com/4496521/109733959-04fcd280-7b75-11eb-922f-3c95d66e7e0d.png" width="400" />

To fix this you can just add a y-translation to the nodes to move it within the viewport, e.g., wrap it in a `<Group top={height}>...`

<img src="https://user-images.githubusercontent.com/4496521/109734010-1c3bc000-7b75-11eb-8165-0a4648d3553f.png" width="400" />

[Here's](https://codesandbox.io/s/compassionate-wescoff-gvzzj?file=/Example.tsx:3104-3108) a modified sandbox.

**Reply by brno32** [reply]
Thanks so much for typing this up! I really appreciate it.

Your tricks worked great 👍
